src/admin-views/tribe-options-general.php

```diff
@@ -127,52 +127,59 @@
 		'type' => 'html',
 		'html' => '</div>',
 	],
 ];
 
 // Add the TOC
 $tec_events_general_toc = [
-	'tec-settings-general-toc-nav-start'       => [
+	'tec-settings-general-toc-nav-start'   => [
 		'type' => 'html',
 		'html' => '<div class="tec-settings-header">',
 	],
-	'tec-settings-general-toc-start'           => [
+	'tec-settings-general-toc-start'       => [
 		'type' => 'html',
 		'html' => '<ul id="tec-events-general-toc" class="tec-events-settings__toc">',
 	],
-	'tec-settings-general-toc-label'           => [
+	'tec-settings-general-toc-label'       => [
 		'type' => 'html',
 		'html' => '<li>' . _x( 'Jump to:', 'Text introducing the table of contents links.', 'the-events-calendar' ) . '</li>',
 	],
-	'tec-settings-general-toc-viewing'         => [
+	'tec-settings-general-toc-viewing'     => [
 		'type' => 'html',
-		'html' => '<li><a href="#tec-settings-general-viewing">' . _x( 'Viewing', 'Viewing table of contents link.', 'the-events-calendar' ) . '</a>',
+		'html' => '<li><a href="#tec-settings-general-viewing">' . _x( 'Viewing', 'Viewing table of contents link.', 'the-events-calendar' ) . '</a></li>',
 	],
-	'tec-settings-general-toc-editing'         => [
+	'tec-settings-general-toc-editing'     => [
 		'type' => 'html',
-		'html' => '<li><a href="#tec-settings-general-editing">' . _x( 'Editing', 'Editing table of contents link.', 'the-events-calendar' ) . '</a>',
+		'html' => '<li><a href="#tec-settings-general-editing">' . _x( 'Editing', 'Editing table of contents link.', 'the-events-calendar' ) . '</a></li>',
 	],
-	'tec-settings-general-toc-maintenance'         => [
+	'tec-settings-general-toc-maintenance' => [
 		'type' => 'html',
-		'html' => '<li><a href="#tec-settings-general-maintenance">' . _x( 'Maintenance', 'Maintenance table of contents link.', 'the-events-calendar' ) . '</a>',
+		'html' => '<li><a href="#tec-settings-general-maintenance">' . _x( 'Maintenance', 'Maintenance table of contents link.', 'the-events-calendar' ) . '</a></li>',
 	],
-	'tec-settings-general-toc-debugging' => [
+	'tec-settings-general-toc-debugging'   => [
 		'type' => 'html',
-		'html' => '<li><a href="#tec-settings-general-debugging">' . _x( 'Debugging', 'Debugging table of contents link.', 'the-events-calendar' ) . '</a>',
+		'html' => '<li><a href="#tec-settings-general-debugging">' . _x( 'Debugging', 'Debugging table of contents link.', 'the-events-calendar' ) . '</a></li>',
 	],
-	'tec-settings-general-toc-end'             => [
+	'tec-settings-general-toc-end'         => [
 		'type' => 'html',
 		'html' => '</ul>',
 	],
-	'tec-settings-general-toc-nav-end'         => [
+	'tec-settings-general-toc-nav-end'     => [
 		'type' => 'html',
 		'html' => '</div>',
 	],
 ];
 
+/**
+ * Allow filtering the Table of Contents bar on the General settings page.
+ *
+ * @var array $tec_events_display_toc Array of items of the TOC.
+ */
+$tec_events_general_toc = apply_filters( 'tec_events_general_settings_toc', $tec_events_general_toc );
```

Comment:
The docblock needs a `@since` and use `@param` folr filters, no need to filter into the variable, than apply the addition. Do it all in one line.